### ISSUE TITLE
Chunk gc cleanup

### ISSUE THREAD

**Issue by boboldehampsink** [spicywebau/craft-neo#866]
### Bug Description

When running GC on a setup with PostgreSQL, the following error can appear:

```
deleting orphaned Neo block structure data ... Error Output: ================ Error: SQLSTATE[HY000]: General error: 7 number of parameters must be between 0 and 65535 The SQL being executed was: DELETE FROM "structures" WHERE ("id" IN (..., ...
```

As you can see, the number of parameters in the delete command exceeds the max of 65535 - so the delete query has to be chunked.

### Steps to reproduce

The same thing has happend and been fixed here: https://github.com/putyourlightson/craft-blitz/issues/639 

### Neo version

2.13.20

### Craft CMS version

3.9.10

### Does this issue involve templating, and if so, is eager-loading used?

This is not a templating issue

**Comment by boboldehampsink** [comment]
@ccchapman ahh, in this case I'm still using Neo 2...

**Comment by ttempleton** [comment]
Yep, looks like I forgot about the Craft 3 version when applying the fix for #773. Thanks for bringing that to my attention - fixed now in 2.13.21.
